builds/misc/templates/build-packages.yaml

```diff
@@ -55,6 +55,10 @@ stages:
               arch: amd64
               os: centos7
               target.iotedged: edgelet/target/rpmbuild/RPMS/x86_64
+            redhat8-amd64:
```

Comment:
In `packages-release.yaml` you have the same identifier but the casing is different (RedHat8-amd64). Does it matter?

Reply:
I don't think so - it's the arch, os, target settings that matter, but it should be consistent.